tools/runs_autopilot.py

```
@@ -975,6 +1182,48 @@ def _add_hash(variant: dict) -> None:
         return any(h in self.completed_hashes for h in hashes if h)
 
+    def _custom_eval_has_results(self, meta: dict) -> bool:
+        if not isinstance(meta, dict):
+            return False
+        suite = str(meta.get("suite") or self.eval_suite)
+        output_dir = meta.get("output_dir")
+        base_path: Optional[Path] = None
+        if output_dir:
+            try:
+                base_path = Path(output_dir)
+            except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)